apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -58,45 +56,26 @@ export function updateDefaultList(listData) {
 }
 
 // Returns the metadata of the list of default sprites in SpriteLab in English
-export function getDefaultListMetadata() {
-  return fetch('/api/v1/animation-library/default-spritelab-metadata')
+export function getDefaultListMetadata(environment = 'production') {
```

Comment:
Tangential question - do we have constants for environment types ('production', 'levelbuilder', etc) that we can reliably use here instead of passing a string directly? (not a huge deal if not)

Reply:
This is a good call! I have a follow-up PR planned for this and will add some constants there for this.